app/models/manageiq/providers/autosde/storage_manager/autosde_client.rb

```diff
@@ -89,6 +89,9 @@ def configure_openapi_client
       config.host = @host
```

Comment:
[this](https://github.com/ManageIQ/manageiq-providers-autosde/pull/52/files#diff-c4817412fc012e61a1492a4a42c07e4cbdc72dcc53a76c759a80a848ceb68415R86) should be changed to `AutosdeOpenapiClient::Configuration`

Reply:
Believe this is working because of the include but that's not really the right way.  I'm going to merge this so we can move forward and I'll put in a follow-up PR to fix this

Reply:
https://github.com/ManageIQ/manageiq-providers-autosde/pull/53